konig-spreadsheet/src/main/java/io/konig/spreadsheet/SourceDataDictionarySheet.java

```diff
@@ -315,18 +317,24 @@ private PropertyConstraint dataDictionaryPropertyConstraint(SheetRow row, SheetC
 			
 		}
 
-//		String snake_case_name = StringUtil.LABEL_TO_SNAKE_CASE(fieldName);
+//		String snake_case_name = StringUtil.LABEL_TO_SNAKE_CASE(fieldName);
 		String snake_case_name = fieldName;
 		String baseURL = settings.getPropertyBaseURL();
 		
 		URI predicate = new URIImpl(concatPath(baseURL, snake_case_name));

```

Comment:
You need to add special logic to address the following requirement...

* When the value in the Derived column is True, the value in the Field column must be treated as an IRI. This means that the code must be prepared to parse the Field value as a CURIE and NOT simply prepend the tabularFieldNamespace as defined in the settings.

Reply:
if I use protected URI iriValue(SheetRow row, SheetColumn column)  method in the BaseSheetProcessor superclass. I am getting "Invalid URI" exception. so add some code to do the same functionality.